crypto/src/tests/bls12381_tests.rs

```diff
@@ -400,3 +403,75 @@ async fn signature_service() {
     // Verify the signature we received.
     assert!(pk.verify(digest.as_ref(), &signature).is_ok());
 }
+
+// Checks if the private keys zeroed out
+#[test]
+fn test_zeroization_priv_key() {
+    let kp = keys().pop().unwrap();
+    let mut sk = kp.private();
```

Comment:
Are we sure `keys` delivers some non-zero bytes in the first place for `sk`? Are we sure that invariant will remain the same forever if somebody modifies how `keys()` works?

Reply:
I've added a deterministic key generation within the test itself.